samples/csharp_dotnetcore/12.customQABot/README.md

```diff
@@ -0,0 +1,89 @@
+﻿# Custom Question Answering
+
+Bot Framework v4 Custom question answering bot sample.
+
+This bot has been created using [Bot Framework][BF], it shows how to create a bot that uses the [Custom question answering feature in Language Service][Quickstart].
+
+The [Custom question answering feature in Language Service][LS] enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured and unstructured documents or editorial content in minutes. In this sample, we demonstrate how to use the Custom question answering feature in Language Service to answer questions based on a FAQ text file used as input.
+
+## Prerequisites
+- This project requires a [Language resource](https://aka.ms/create-language-resource) with Custom question answering enabled.
+
+### Configure knowledge base of the project
+- Follow instructions [here][Quickstart] to create a Custom question answering project. You will need this project's name to be used as `ProjectName` in [appsettings.json](appsettings.json).
+- Visit [Language Studio][LS] and open created project.
+- Go to `Edit knowledge base` -> Click on `...` -> Click on `Import questions and answers` -> Click on `Import as TSV`.
+- Import [SampleForCQA.tsv](CognitiveModels/SampleForCQA.tsv) file.
+- You can test your knowledge base by clicking on `Test` option.
+- Go to `Deploy knowledge base` and click on `Deploy`.
+
+### Connect your bot to the project.
+Follow these steps to update [appsettings.json](appsettings.json).
+- In the [Azure Portal][Azure], go to your resource.
+- Go to `Keys and Endpoint` under Resource Management.
+- Copy one of the keys as value of `LanguageEndpointKey` and Endpoint as value of `LanguageEndpointHostName` in [appsettings.json](appsettings.json).
+- `ProjectName` is the name of the project created in [Language Studio][LS].
+
+## To try this sample
+
+- Install the Bot Framework Emulator version 4.14.0 or greater from [here][BFE]
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-samples.git
+    ```
+
+- In a terminal, navigate to `samples/csharp_dotnetcore/12.customQABot`
+- Run the bot from a terminal or from Visual Studio, choose option A or B.
+
+  A) From a terminal
```

Comment:
Lines 39-56 should be indented appropriately to render as sub-points of Line 37.

Reply:
Hi, I checked and the lines seem to be rendered fine in github:

![image](https://user-images.githubusercontent.com/12264946/165850034-9f4f4a08-9759-4291-a5e8-043738684a46.png)